src/packages/cli/fixtures/custom generator/prisma/schema.prisma

```diff
@@ -5,11 +5,11 @@ generator min {
 
 datasource db {
   provider = "sqlite"
-  url      = env("DATABASE_URL")
```

Comment:
Should we have a test where the env var is actually not set?

Reply:
Revert in https://github.com/prisma/prisma/pull/7793